src/ConnectModal.tsx

```diff
@@ -98,7 +99,7 @@ function deviceList(
             aria-label={d.label}
           >
             {t.isWireless && (
-              <SignalIcon className="w-4 justify-center content-center h-full" />
+              <Radio className="w-4 justify-center content-center h-full" />
```

Comment:
Is there a reason you're not using the `bluetooth` icon from lucide? https://lucide.dev/icons/bluetooth

Reply:
No other reason than mimicking the equivalent Hero icon. I also didn't test this part of the app, so I wanted to keep it as similar as possible

Reply:
Please switch to the BT icon, the hero icon was out of lack of choice.